yb_migrate/cmd/exportData.go

```diff
@@ -173,7 +176,7 @@ func exportDataOffline() bool {
 	utils.WaitGroup.Wait() //waiting for the dump to complete
 
 	if ctx.Err() != nil {
-		fmt.Printf("ctx error(exportData.go): %v\n", ctx.Err())
+		utils.ErrExit(fmt.Sprintf("ctx error(exportData.go): %v", ctx.Err()))
```

Comment:
Drop this call to `utils.ErrExit()`. This changes the existing behaviour. In case of error, this function is expected to return `false`. But `utils.ErrExit()` exits the entire process.